lib/timescaledb/schema_dumper.rb

```diff
@@ -5,19 +5,12 @@ module Timescaledb
   module SchemaDumper
     def tables(stream)
       super # This will call #table for each table in the database
-      views(stream) unless defined?(Scenic) # Don't call this twice if we're using Scenic
```

Comment:
debugging this part, I  discover the previous  `super`  is also calling the  `views`.